src/cli.ts

```diff
@@ -89,13 +93,20 @@ const splitDirection = args["--split-vertically"]
   ? "horizontally"
   : undefined;
 
+let optionsFromFile: InitialOptions = {};
+try {
+  optionsFromFile = JSON.parse(fs.readFileSync("./.newsh.json").toString());
+} catch (e) {}
```

Comment:
In case the `.newsh.json` file is an invalid json, the user will have hard time debugging this as we don't throw an error.

I think we can use something like https://github.com/davidtheclark/cosmiconfig to handle those edge cases.

In any case, I would prefer adding a `config.ts` file when we can handle all of this.

We also have a node API available, and it would make sense for the config to support it as well.